client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -5,10 +5,27 @@ import Filters from '../../components/Filter/Filter';
 import NavBar from '../../components/Navbar/Navbar';
 import React from 'react';
 import ProfilePage from '../ProfilePage/ProfilePage';
+import Map from 'pigeon-maps';
+import Marker from 'pigeon-marker';
+import Overlay from 'pigeon-overlay';
 
 export default function FiltersPage() {
   const [listOrMap, setListOrMap] = React.useState(true);
```

Comment:
can we change this to a better name like `listMode`? it doesn't make much sense to say that `listOrMap` is `true`